sdk/storage/azure-storage-blob/tests/test_block_blob_async.py

```diff
@@ -466,7 +466,7 @@ def return_response(resp, _, headers):
         resp, headers = await blob.stage_block(0, 'block 0', cls=return_response)
 
         # Assert
-        self.assertEqual(201, resp.status_code)
+        self.assertEqual(201, resp.http_response.status_code)
```

Comment:
this will be breaking change to customers right

Reply:
Offline: this is not a breaking change.